src/EnergyPlus/WindowManager.cc

```diff
@@ -569,10 +579,18 @@ namespace WindowManager {
 			if ( Construct( ConstrNum ).WindowTypeEQL ) continue; //skip Equivalent Layer Fenestration
 			//handling of optical properties
 
+			for ( IPhi = 1; IPhi <= 10; ++IPhi ) {
```

Comment:
@mjwitte 10 is OK for data type as Spectral as initial guess. If Data Type = SpectralAndAngle, the values will be revised based on the number of angles from inputs.